websocket/frontend/src/todolist/Todoform.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+class TodoForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      task: "",
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleTaskChange = this.handleTaskChange.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.formSubmit(event.target);
+  }
+
+  formSubmit(formData) {
+    const data = new FormData(formData);
+    const obj = {
+      method: 'POST',
+      mode: 'cors',
+      body: data,
+    };
+    fetch(this.props.apiURL, obj)
+    .then(res => res.json())
+    .then(resJson => this.props.updateTodoList(resJson));
```

Comment:
```suggestion
    fetch(this.props.apiURL, obj)
      .then(res => res.json())
      .then(resJson => this.props.updateTodoList(resJson));
```
doing it this way makes it easier to see the `.then` belongs to `fetch`